bricks/_common/arm_none_eabi.mk

```diff
@@ -637,6 +638,11 @@ $(BUILD)/u-boot.bin:
 	$(Q)mkdir -p $(dir $@)
 	$(Q)curl -sL -o $@ https://github.com/pybricks/u-boot/releases/download/pybricks/v2.0.1/u-boot.bin
 	$(Q)echo "86ddad84f64d8aea85b4315fc1414bdec0bb0d46c92dbd3db45ed599e3a994cb  $@" | sha256sum -c --strict
+$(BUILD)/pru_ledpwm.bin:
+	$(ECHO) "Downloading pru_ledpwm.bin"
+	$(Q)mkdir -p $(dir $@)
+	$(Q)curl -sL -o $@ https://github.com/pybricks/pybricks-pru/releases/download/v0.0.1/pru_ledpwm.bin
```

Comment:
I would already call it v1.0.0.  (I no longer believe in https://0ver.org/) :laughing: